src/ace/managers/memory.c

```diff
@@ -190,10 +190,7 @@ void *_memAllocDbg(
 			"[MEM] ERR: couldn't allocate %lu bytes! (%s:%u)\n",
 			ulSize, szFile, uwLine
 		);
-		logWrite(
-			"[MEM] Peak usage: CHIP: %lu, FAST: %lu\n",
-			s_ulChipPeakUsage, s_ulFastPeakUsage
-		);
+		memLogPeak();
```

Comment:
Should this also stay inlined to avoid a function call?

Reply:
it's only for debug builds, so I think a fn call is okay - it won't be fast anyway ;)

Reply:
That's what I hoped, good to hear. Did I see that CI failed on this, false alarm?